src/components/Modal/Confirm.vue

```diff
@@ -16,18 +17,22 @@ const props = defineProps({
 
 const emit = defineEmits(['reload', 'close']);
 
-const { send } = useClient();
+const { send, sign } = useClient();
+const { web3 } = useWeb3();
+const format = getChoiceString;
 
 const loading = ref(false);
+
 const symbols = computed(() =>
   props.strategies.map(strategy => strategy.params.symbol)
 );
-
-const format = getChoiceString;
+const walletConnectType = computed(() => web3.value.walletConnectType);
 
 async function handleSubmit() {
   loading.value = true;
-  await send(props.space.id, 'vote', {
+  const isSafe = walletConnectType.value === 'Gnosis Safe Multisig';
```

Comment:
This should be done in useClient, we want the same logic for create a proposal and update space settings

Reply:
Ok makes sense